### DISCUSSION TITLE
When rebasing a PR, renovate does not uncheck rebase option after rebase is complete

### DISCUSSION THREAD

**Discussion by kolea2** [renovatebot/renovate#21074] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the bug

PR example: https://github.com/googleapis/java-bigtable/pull/1668

I checked the box in the PR description to rebase the PR - it looks like renovate rebased, but never unchecked the box once complete. This caused it to rebase ~once every minute. This used to work fine, so this might be due to a recent change. Unchecking the box manually stopped it from rebasing continually.

Let me know if you need any other information, thanks for the help!

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

I have read the minimal reproductions documentation and linked to such a repository in the bug description

**Comment by rarkins** [comment]
Note: this is in Forking Renovate, I will see if it can be reproduced

**Reply by rarkins** [reply]
Issue cannot be reproduced, so I have converted it to a discussion

**Comment by rarkins** [comment]
Could not reproduce on https://github.com/rarkins/rebase1/pull/2

I ticked the rebase checkbox, and it rebased + cleared the box as expected. The referenced PR from googleapis has been merged now so unfortunately cannot be used to debug.

@kolea2 please let us know if the problem reoccurs. There must have been something specific about the repo or the particular PR which caused this, and without a live occurrence it's to hard to guess why it was.

**Reply by kolea2** [reply]
Thank you for the investigation, @rarkins! I tried again with a different PR and it worked properly - I will keep an eye out and add any new details to this discussion.

**Comment by kolea2** [comment]
@rarkins here's another example https://github.com/googleapis/java-bigtable/pull/1618

I manually unchecked the box so that it would stop rebasing and kicking off the presubmit checks, but let me know if you need anything else.

**Reply by rarkins** [reply]
Could you try logging into https://developer.mend.io and seeing if the log for that recent run is there? If so, please copy/paste it into a gist and link to it from here.

I _can_ gain access to the logs myself if it's not possible, but it involves a lot of VPN'ing and 2FA'ing and so on.

**Comment by rarkins** [comment, marked as answer]
I was able to see this happen myself, have raised a bug, so let's move conversation there: https://github.com/renovatebot/renovate/issues/21144

**Reply by kolea2** [reply]
sorry for the delay but thank you for taking a look @rarkins!

**Reply by rarkins** [reply]
As of right how this should be fixed in both Renovate and Forking Renovate apps
